pkg/smokescreen/acl_loader_v1.go

```diff
@@ -22,38 +22,41 @@ type EgressAclConfig struct {
 	Default  *EgressAclRule
 }
 
-func (ew *EgressAclConfig) Decide(fromService string, toHost string) (EgressAclDecision, error) {
+func (ew *EgressAclConfig) Decide(fromService string, toHost string) (EgressAclDecision, bool, error) {
+	var action EgressAclDecision
+
 	rule := ew.ruleForService(fromService)
 	if rule == nil {
-		return 0, fmt.Errorf("unknown role: '%s'", fromService)
```

Comment:
`ruleForService` returns nil only if no default rule is configured.  Do we care about distinguishing this case from the default rule being explicitly set to "deny"?  I'm generally not thrilled with adding a special case to the enum and also another return parameter.

Reply:
Agreed!